### ISSUE TITLE
Loading logs do not stop after click to button show all logs. 

### ISSUE THREAD

**Issue by 2Abendsegler** [2Abendsegler/GClh#825]
**Describe the bug**
Loading logs do not stop after click to button show all logs. 

**To Reproduce**
1. Go to https://www.geocaching.com/geocache/GC2FC29_tobotronc-3
2. Wait until the loading icons in VIP Lists are gone
3. Go to the Logs and click the button show all logs or an icon for a section
4. Scroll to the end and see the loading icon

It could be that it is the loading icon of GS.

**Comment by Ruko2010** [comment]
> It could be that it is the loading icon of GS.

it is 😉. If you open the "code" you see, that if you scroll down more logs are added to the original "cache_logs_table"

**Comment by Ruko2010** [comment]
I tried several things, but I can't get the "scroll" Event added by GC to unbind.
![image](https://user-images.githubusercontent.com/2239440/45869468-d1ed6c80-bd88-11e8-9ef2-66a57e3e21e0.png)

The first window scroll Event is ours, the other two should be removed. If I remove them live in chrome, GC stopped loading logs, but I can't remove them with GClh. Here are some things I tried before adding our own Event:
![image](https://user-images.githubusercontent.com/2239440/45870081-60aeb900-bd8a-11e8-9d75-e4c842e8a509.png)
